recipe/conda_build_config.yaml

```diff
@@ -172,8 +172,6 @@ pin_run_as_build:
     max_pin: x
   dbus:
     max_pin: x
-  expat:
-    max_pin: x.x
```

Comment:
Should we review our packager that have run exports and still have this `pin_run_as_build` here? It is a tedious job but seems that it would save us a few PRs in the future.